### PR TITLE
Test scalar multiplication/division with int and zero

### PR DESCRIPTION
**Pull request by nbraud**

### CODE REVIEW

File: tests/test_scalar_multiplication.py [hunk 1]
```diff
@@ -50,3 +51,23 @@ def test_scalar_inverse(x: Vector, scalar: float):
...
+    assume(scalar != 0)
+    assert x / scalar == x / float(scalar)
+
+
+@given(x=vectors())
+def test_division_by_zero(x: Vector):
+    with raises(ZeroDivisionError):
+        x / 0
+
+    with raises(ZeroDivisionError):
```
Comment (astronouth7303) [inline tests/test_scalar_multiplication.py]: Why twice?

File: tests/test_scalar_multiplication.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from math import isclose
 
-from hypothesis import assume, given
+from hypothesis import assume, given, strategies as st
+from pytest import raises  # type: ignore
```
Comment (astronouth7303) [inline tests/test_scalar_multiplication.py]: :sigh: what complains about this?

File: tests/test_scalar_multiplication.py (hunk 1 above)
Comment (nbraud) [reply tests/test_scalar_multiplication.py]: With integer `0` and floating-point `0.`:
```
>>> type(0), type(0.)
(int, float)
```

File: tests/test_scalar_multiplication.py (hunk 2 above)
Comment (nbraud) [reply tests/test_scalar_multiplication.py]: `mypy` simply doesn't have an interface definition for `pytest`, which is why all `pytest` imports need to have a `type: ignore` annotation.

File: tests/test_scalar_multiplication.py (hunk 1 above)
Comment (astronouth7303) [reply tests/test_scalar_multiplication.py]: Oh! Missed the dot.

**Pull request merged**